main.py

```diff
@@ -1,10 +1,41 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+
+    for nickname, info in players_data.items():
+        race = info["race"]
+        new_race, _ = Race.objects.get_or_create(
+            name=race["name"],
+            description=race["description"]
+        )
+
+        for skill in race["skills"]:
+            Skill.objects.get_or_create(
+                race=new_race,
+                name=skill["name"],
+                bonus=skill["bonus"]
+            )
+
+        guild = info.get("guild", None)
```

Comment:
No need to pass None. It's the default value that is used when key is not present in the dictionary